src/app/key-concepts/natural-capital-and-climate-change/page.tsx

```diff
@@ -0,0 +1,197 @@
+import Footer from "@/components/footer";
```

Comment:
I have not reviewed the code of this page but I think all the comments I left on the other can be applied here.